# A message containing all of the objects that were detected by the YOLO algorithm, alongside their respective timestamps

Header header
yolov3_bounding_box[] bb_data
